funding/manager.go

```diff
@@ -2036,55 +2046,61 @@ func (f *Manager) funderProcessAcceptChannel(peer lnpeer.Peer,
 	// required confirmations, and also the set of channel constraints
 	// they've specified for commitment states we can create.
 	resCtx.reservation.SetNumConfsRequired(uint16(minDepth))
-	channelConstraints := &channeldb.ChannelConstraints{
-		DustLimit:        msg.DustLimit,
+	bounds := channeldb.ChannelStateBounds{
 		ChanReserve:      msg.ChannelReserve,
 		MaxPendingAmount: msg.MaxValueInFlight,
 		MinHTLC:          msg.HtlcMinimum,
 		MaxAcceptedHtlcs: msg.MaxAcceptedHTLCs,
-		CsvDelay:         msg.CsvDelay,
+	}
+	rParams := channeldb.CommitmentParams{
```

Comment:
nit: what is this `r`? should we name is `cParams` or just `params` since in this context the params can only mean commitment params.

Reply:
Ah this is leftover from a previous revision. I'll address that. It used to stand for "renderingParameters" since these are the values that allow us to "render" abstract channel state into a concrete commitment transaction. However, at @bhandras' request I opted for less esoteric naming.